charts/gsp-cluster/charts/gsp-monitoring/templates/sli-dashboard.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: '{{ .Release.Name }}-grafana-sli-dashboard'
```

Comment:
Is there a reason this is quoted? I'm just curious...

Reply:
The reason this is quoted is because I copypasted from grafana-concourse-dashboard.yaml.

The reason *that* one is quoted is.. I think to make my emacs flycheck happy, which gets grumpy about the curly brackets for some reason.  I should probably fix my emacs to not care though.